flashlight-firmware/anduril2/ToyKeeper/spaghetti-monster/anduril/aux-leds.c

```diff
@@ -26,8 +26,13 @@
 #if defined(USE_INDICATOR_LED) && defined(TICK_DURING_STANDBY)
 // beacon-like mode for the indicator LED
 void indicator_blink(uint8_t arg) {
+    #ifdef USE_VOLTAGE_LOW_BLINKING_INDICATOR
```

Comment:
where is this being defined?

Reply:
This flag should be defined in the specific flashlight's header file. e.g., cfg-blf-lantern.h

At least, this seems proper since that's where all the other similar flags are defined.

Reply:
I tried to follow ToyKeeper's template of how to add these definitions and config options: commented out if the default is disabled, a comment for what it does, so it's easy for someone to see it and take the action.  otherwise, a new user would need to stumble upon this block to know to enable it.  https://github.com/mkong1/anduril/pull/13/files is an example of a similar change I made to add a config option to make the light unlock to `off` state instead of `on` state.

Reply:
I've added the commented out define flag with some user notes in my recent commit 7addbda